### PR TITLE
BUG-119024 changes required to make it work on 2.6 clusters

### PR DESCRIPTION
**Pull request by mukund-thakur**
Please don't merge. Still need to test end to end once.

### CODE REVIEW

File: dp-cluster-setup-utility.py [hunk 1]
```diff
@@ -433,11 +433,29 @@ def enable_trusted_proxy_for_beacon(self):
...
+
+
+  def enable_trusted_proxy_for_ambari_2_6(self):
+    print 'Enabling Knox Trusted Proxy Support in Ambari 2.6 '
+    ambari_host = self.internal_host
+    knox_user = self.cluster.knox_user()
+    knox_host = self.cluster.knox_host()
+    if ambari_host != knox_host :
+      print "Ambari host and Knox host are not same." \
+            "Please run the ambari-server setup-trusted-proxy in Ambari host as a prerequisite."
```
Comment (shwethags) [inline dp-cluster-setup-utility.py]: Can we also tell the user what the values should be for the command

File: dp-cluster-setup-utility.py [hunk 2]
```diff
@@ -1124,7 +1155,7 @@ def stack_supported(self):
 
   def hdp_supported_version(self):
     stack = self.ambari.installed_stack()
-    return stack.name == 'HDP' and stack.version.startswith('3.1')
+    return stack.name == 'HDP' and ( stack.version.startswith('3.1') or stack.version.startswith('2.6') )
```
Comment (shwethags) [inline dp-cluster-setup-utility.py]: Shouldn't it be 2.6.5 specifically?

Comment (mukund-thakur) [reply dp-cluster-setup-utility.py]: Ambari stack version return 2.6 only.

**Comment by mukund-thakur** [comment]
Tested. Can be merged now.

**Pull request merged**